doc/explanation/clustering.md

```diff
@@ -37,86 +48,101 @@ Automatic roles are assigned by LXD itself and cannot be modified by the user.
+By default, LXD replicates images on as many cluster members as there are database members.
+This typically means up to three copies within the cluster.
 
-Those typically cover:
+You can increase that number to improve fault tolerance and the likelihood of the image being locally available.
+To do so, set the [`cluster.images_minimal_replica`](server) configuration.
+The special value of `-1` can be used to have the image copied to all cluster members.
 
-- Source device for a storage pool (leaving empty would create a loop)
-- Name for a ZFS zpool (defaults to the name of the LXD pool)
-- External interfaces for a bridged network (empty would add none)
-- Name of the parent network device for managed `physical` or `macvlan` networks (must be set)
+## Cluster groups
 
-It's possible to lookup the questions ahead of time (useful for scripting) by querying the `/1.0/cluster` API endpoint.
-This can be done through `lxc query /1.0/cluster` or through other API clients.
+In a LXD cluster, you can add members to cluster groups.
+You can use these cluster groups to launch instances on a cluster member that belongs to a subset of all available members.
+For example, you could create a cluster group for all members that have a GPU and then launch all instances that require a GPU on this cluster group.
 
-## Images
+By default, all cluster members belong to the `default` group.
 
-By default, LXD will replicate images on as many cluster members as you have database members.
-This typically means up to 3 copies within the cluster.
+See {ref}`cluster-groups` and {ref}`cluster-target-instance` for more information.
 
-That number can be increased to improve fault tolerance and likelihood of the image being locally available.
+(clustering-assignment)=
+## Automatic assignment of instances
 
-The special value of "-1" may be used to have the image copied on all nodes.
+In a cluster setup, each instance lives on one of the cluster members.
+When you launch an instance, you can target it to a specific cluster member, to a cluster group or have LXD automatically assign it to a cluster member.
 
-You can disable the image replication in the cluster by setting the count down to 1:
+By default, the automatic assignment picks the cluster member that has the lowest number of instances.
+If several members have the same amount of instances, one of the members is chosen at random.
 
-```bash
-lxc config set cluster.images_minimal_replica 1
-```
+However, you can control this behavior with the [`scheduler.instance`](cluster-member-config) configuration option:
 
-## Cluster groups
+- If `scheduler.instance` is set to `all` for a cluster member, this cluster member is selected for an instance if:
+
+   - The instance is started without `--target` and the cluster member has the lowest number of instances.
+   - The instance is targeted to run on this cluster member.
```

Comment:
I think "exist" or "live" would be better than "run" here for the same reason as above.